modules/nback.llvm/llvm.bal

```diff
@@ -56,15 +56,23 @@ public distinct class BasicBlock {
     private final string label;
     private final string[] lines;
 
-    function init(string label, Function func) {
+    function init(string label, Function func, boolean ignoreLabel = false) {
         self.label = label;
         self.func = func;
-        self.lines = [label + ":"];
+        if ignoreLabel {
```

Comment:
`ignoreLabel` obscure the reason we have it here, it would be clear it was `isEntry`

Reply:
Fixed